CHANGELOG.md

```diff
@@ -4,10 +4,35 @@ This changelog is a work in progress and may contain notes for versions which ha
 
 ## v10.0.0
 
-### Features ✅
+### Breaking changes 🛠
 
--   Upgraded to Go 1.14, which contains several WebAssembly performance improvements [#815](https://github.com/0xProject/0x-mesh/pull/815).
 -   Reject orders that have taker addresses that are non-zero and are not whitelisted. [#896](https://github.com/0xProject/0x-mesh/pull/896).
+-   Removed support for Coordinator orders. [#895](https://github.com/0xProject/0x-mesh/pull/895).
+-   The JSON-RPC is no longer supported by Mesh nodes. Only a GraphQL API is supported. [#858](https://github.com/0xProject/0x-mesh/pull/858)
```

Comment:
If the breaking changes are sorted by impact, this should probably be up top.

Reply:
I was sorting them by the PR number, so I'll leave this for now.